pop/README.md

```diff
@@ -0,0 +1,119 @@
+# Proof of Personhood
+
+pop command is used by one of the two groups of users:
+
+- organizers (org)
+- attendees (attendee)
+
+Each group has its own set of commands to start, finalize and use the keys.
+
+## Organizers
+
+### Link
+
+First every organizer has to link to his conode:
+
+```bash
+pop org link 127.0.0.1:2002
+# Read pin from command line
+pop org link 127.0.0.1:2002 PIN
+```
+
+### Store the configuration
+
+Now each organizer registers the description that they got together:
+`description.toml` - example:
```

Comment:
Each organizers should assemble their identity generated when first running the conode. The description.toml contains the list of identities of all organizers and some additional information about the party.

(stg like that ;) )

Reply:
Each + singular: Each organizer ... his identity...